tests/testthat/test-use_extendr.R

```diff
@@ -1,4 +1,6 @@
 test_that("use_extendr() sets up extendr files correctly", {
+  skip_if(!requireNamespace("usethis", quietly = TRUE))
```

Comment:
Should we just embrace `{usethis}` at this point? We do so much ceremony to check if the package is available, but we rely on its functionality a lot.

Reply:
I believe this is a common way to write tests for functions that call optional dependencies.

Reply:
Sorry, `skip_if_not_installed` should be used here.